vision/spring-framework/src/main/resources/static/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Google Cloud Vision API with Spring code sample</title>
+</head>
+<body>
+
+<div>
+    <h1>Image Label Annotations</h1>
+    <p>Returns labels classifying the content of the image:</p>
+    <form action="/extractLabels">
+        Web URL of image to analyze: <input type="text" name="imageUrl"/> <input type="submit"/>
+    </form>
+</div>
+
+<div>
+    <h1>Text Extraction</h1>
+    <p>Read and extract the text from the image:</p>
+    <form action="/extractText">
+        Web URL of image to analyze: <input type="text" name="imageUrl"/> <input type="submit" />
```

Comment:
Same here, can you prefill with something from the web?

Reply:
Done.